client/lib/plans/constants.js

```diff
@@ -578,12 +625,24 @@ export const PLANS_LIST = {
 			FEATURE_SEO_PREVIEW_TOOLS,
 			FEATURE_ALL_PREMIUM_FEATURES
 		] ),
+		getNewSignupFeatures: () => [
+			FEATURE_UNLIMITED_PREMIUM_THEMES,
+			FEATURE_OFFSITE_BACKUP_VAULTPRESS_REALTIME,
+			FEATURE_SEO_PREVIEW_TOOLS,
+			FEATURE_CONCIERGE_SETUP,
+			FEATURE_ALL_PREMIUM_FEATURES,
+		],
 		getBillingTimeFrame: () => i18n.translate( 'per month, billed monthly' ),
 		getSignupBillingTimeFrame: () => 'per month', //PLANS A/B TEST: Translate if test passes
 	}
 };
 
 export const FEATURES_LIST = {
+	[ FEATURE_BLANK ]: {
+		getSlug: () => FEATURE_BLANK,
+		getTitle: () => '', //PLANS A/B TEST: Translate if test passes
```

Comment:
We won't actually need to translate this one, so we can remove this comment, right?